arbitrum-docs/how-arbitrum-works/inside-anytrust.md

```diff
@@ -8,48 +8,48 @@ AnyTrust is a variant of Arbitrum Nitro technology that lowers costs by acceptin
 
-A Keyset contains
+A Keyset contains:
 
 - the number of Committee members, and
 - for each Committee member, a BLS public key, and
 - the number of Committee signatures required.
 
 Keysets are identified by their hashes.
 
-An L1 KeysetManager contract maintains a list of currently valid Keysets. The L2 chain's Owner can add or remove Keysets from this list. When a Keyset becomes valid, the KeysetManager contract emits an L1 Ethereum event containing the Keyset's hash and full contents. This allows the contents to be recovered later by anyone, given only the Keyset hash.
+The SequencerInbox contract maintains `dasKeySetInfo` mapping with embedded [access controls](https://github.com/OffchainLabs/nitro-contracts/blob/fbbcef09c95f69decabaced3da683f987902f3e2/src/bridge/SequencerInbox.sol#L750-L778) for adding new keysets and invalidating existing ones. Upon addition, a L1 system contract [event](https://github.com/OffchainLabs/nitro-contracts/blob/fbbcef09c95f69decabaced3da683f987902f3e2/src/bridge/ISequencerInbox.sol#L197-L201) containing the Keyset's hash and full contents is emitted and processed by the L2 chain during [validation](https://github.com/Layr-Labs/nitro/blob/eigenda-v3.2.1/arbstate/daprovider/util.go#L194).
 
-Although the API does not limit the number of Keysets that can be valid at the same time, normally only one Keyset will be valid.
+Although the API does not limit the number of active Keysets that can be valid at the same time, normally only one Keyset will be valid.
 
 ## Data Availability Certificates
 
-A central concept in AnyTrust is the Data Availability Certificate (hereafter, a "DACert"). A DACert contains:
+A central concept in AnyTrust is the Data Availability Certificate (hereafter, a "DACert"). A [DACert](https://github.com/Layr-Labs/nitro/blob/eigenda-v3.2.1/arbstate/daprovider/util.go#L239-L246) contains:
 
 - the hash of a data block, and
 - an expiration time, and
-- proof that N-1 Committee members have signed the (hash, expiration time) pair, consisting of
+- proof that N-1 Committee members have signed the (hash, expiration time) pair, consisting of:
   - the hash of the Keyset used in signing, and
   - a bitmap saying which Committee members signed, and
-  - a BLS aggregated signature (over the BLS12-381 curve) proving that those parties signed.
+  - an aggregated BLS signature (over the BLS12-381 curve) proving that those parties signed.
 
-Because of the 2-of-N trust assumption, a DACert constitutes proof that the block's data (i.e., the preimage of the hash in the DACert) will be available from at least one honest Committee member, at least until the expiration time.
+Because of the `1-of-N` trust assumption, a DACert constitutes proof that the batch data (i.e., the preimage of the hash in the DACert) will be available from at least one honest Committee member, at least until the expiration time.
```

Comment:
```suggestion
Because of the `2-of-N` trust assumption, a DACert constitutes proof that the block's data (i.e., the preimage of the hash in the DACert) will be available from at least one honest Committee member, at least until the expiration time.
```

We are using the same language throughout the rest of the page, so I wouldn't change this instance alone. If we want to change "data block" to "batch" we would have to do so in all instances.